File: OstjApp/OstjApi/Models/Person.cs
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Ostj.Constants;

namespace OstjApi.Models
{
    [Index(nameof(Email), IsUnique = true)]
    public class Person
    {
        public int Id { get; set; }
        [MaxLength(Constants.MaxTitleLength)]
        public string Name { get; set; } = string.Empty;
        [MaxLength(Constants.MaxNameLength)]
        public string Email { get; set; } = string.Empty;
        [MaxLength(20)]
        public string? Phone { get; set; }
        [MaxLength(Constants.MaxNameLength)]
        public EnrollmentType EnrollmentType { get; set; } = EnrollmentType.NotEnrolled;
        public IList<Profile> Profiles { get; set; } = [];
    }
}

File: resume-matching/src/main/java/com/ostj/dataentity/Person.java
package com.ostj.dataentity;

import java.util.ArrayList;
import java.util.List;

public class Person {
    public int id = -1;
    public String name = "";
    public String email = "";
    public String phone = "";
    public String city = "";
    public String state = "";
    public List<Profile> profiles;

    public Person(){
        this.profiles = new ArrayList<Profile>();
    }

    public String toString(){
        return String.format("Id=%d", id )+", Name="+name+", Email="+email+", Phone="+phone+", City="+city+", State="+state+", resumes count="+profiles.size();
    }
}
